node-notes-cli/delete-note.js

```diff
@@ -0,0 +1,21 @@
+import { readFileSync } from 'node:fs'
+import { writeFile } from 'node:fs/promises';
+
+export const remove = (id) => {
+  const data = readFileSync('data.json', 'utf8');
+  const parsedData = JSON.parse(data);
+  const numberProp = process.argv[3];
+
+  if (parsedData.notes[numberProp]) {
+    delete parsedData.notes[numberProp];
+  } else {
+    console.error('ID does not exist!');
```

Comment:
You will log the error but then continue executing this function. That's not usually what you want.

You do the same thing below. Fix that one, too.